tests/thread_float/tests/01-run.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 Otto-von-Guericke-Universität Magdeburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+
+import sys
+from testrunner import run
+
+
+def assertAlmostEqual(first, second, delta=0.05):
+    assert first + delta > second
+    assert first - delta < second
```

Comment:
That's a useful function that could be used elsewhere I think. Maybe add it in testrunner (any opinion for a better place) and import it from there ?

Reply:
See https://github.com/RIOT-OS/RIOT/pull/16901#discussion_r717918329. Maybe then it is better to implement this test as a `PexpectTestCase` instead.

Reply:
Since the function is two lines, it might be easier for the occasional Python reader to just look through the code instead of having to consult the documentation.